cdr_schemas/features/polygon_features.py

```diff
@@ -50,44 +52,67 @@ class PolygonFeatureCollection(BaseModel):
     """
 
     type: GeoJsonType = GeoJsonType.FeatureCollection
-    features: Optional[List[PolygonFeature]]
+    features: Optional[List[PolygonFeature]] = None
 
 
 class MapUnit(BaseModel):
     """
     Map unit information for legend item.
     """
 
-    age_text: Optional[str]
-    b_age: Optional[float]
-    b_interval: Optional[str]
-    lithology: Optional[str]
-    name: Optional[str]
-    t_age: Optional[float]
-    t_interval: Optional[str]
-    comments: Optional[str]
+    # TODO Add details on what these fields are / are we keeping them
+    age_text: Optional[str] = None
+    b_age: Optional[float] = None
+    b_interval: Optional[str] = None
+    lithology: Optional[str] = None
+    name: Optional[str] = None
+    t_age: Optional[float] = None
+    t_interval: Optional[str] = None
+    comments: Optional[str] = None
 
 
-class PolygonLegendAndFeauturesResult(BaseModel):
+class PolygonLegendAndFeatureResult(BaseModel):
     """
-    Polygon legend item metadata along with associated polygon features found.
+    Polygon map unit metadata along with associated polygon segmentation found.
     """
 
     id: str = Field(description="your internal id")
-    crs: str = Field(description="values={CRITICALMAAS:pixel, EPSG:*}")
-    cdr_projection_id: Optional[str] = Field(
-        description="""
-                        A cdr projection id used to georeference the features
-                    """
+
+    # Legend Fields
+    label: Optional[str] = Field(default=None, description="Label of the map unit")
+    abbreviation: Optional[str] = Field(
+        default=None, description="Abbreviation of the map unit label."
+    )
+    description: Optional[str] = Field(
+        default=None, description="Description of the map unit"
     )
-    map_unit: Optional[MapUnit]
-    abbreviation: Optional[str]
     legend_bbox: Optional[List[Union[float, int]]] = Field(
-        description="""The extacted bounding box of the legend item.
-        Column value from left, row value from bottom."""
+        default=None,
+        description="The rough 2 point bounding box of the map units label.",
```

Comment:
Maybe add in example like [x_min, y_min, x_max, y_max]
Since the order of these matter and I have seen other formats for bbox that are not this exact one.